style.css

```diff
@@ -310,3 +334,142 @@ li {
   border-radius: 30px;
   margin: 20px auto 0 auto;
 }
+
+/* MEDIA QUERIES */
+
+@media screen and (min-width: 992px) {
+  .header {
+    padding: 0 32px;
+  }
+
+  .menu .humbeger {
+    display: none;
+  }
+
+  .menu__links {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .menu__links-item {
+    margin-right: 25px;
+  }
+
+  .showcase {
+    background-image: url("./img/header-shapes-desktop.png");
+    border-bottom-left-radius: 110px;
+    height: 85vh;
+    padding-top: 100px;
+  }
+
+  .showcase__presentation {
+    width: 700px;
+    margin: 0 auto;
+  }
+
+  .works {
+    grid-template-columns: repeat(2, 1fr);
+    width: 70%;
+    height: 460px;
+    margin: 70px auto;
+  }
+
+  .works-col-2-1 {
+    display: flex;
+    flex-direction: row;
+    width: 70%;
+    padding: 16px;
+    background: #fff;
+    border-radius: 8px;
+    margin: 50px auto;
+  }
+
+  .works-col-2-1 .work__description {
+    order: -1;
+  }
+
+  .works__image {
+    margin-right: 30px;
+  }
+
+  .work__description > h2 {
+    margin-top: 30px;
+    width: 400px;
+  }
+
+  .short_desc {
+    margin-bottom: 30px;
+  }
+
+  .about-me {
+    grid-template-columns: repeat(2, 1fr);
+    width: 100%;
+    padding: 20px 150px;
+    height: 500px;
+    border-top-right-radius: 100px;
+  }
+
+  .programming-lang-container {
+    display: flex;
+    width: 100px;
+  }
+
+  .programming-lang {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 5px;
+    margin-top: 5px;
+    margin-right: 10px;
+    height: 76px;
+  }
+
+  .programming-lang img {
+    width: 40px;
+    height: 40px;
+  }
+
+  .prog-languages h3 {
+    margin-bottom: 1px;
+  }
+
+  .about-me__tools {
+    margin-bottom: 0;
+  }
+
+  .contact-form {
+    display: flex;
+    flex-direction: column;
+    background-image: url("/img/contact-form-desk-bg.png");
+    border-top-left-radius: 100px;
+  }
+
+  .contact-form__intro p {
+    width: 500px;
+  }
+
+  .form-group {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
```

Comment:
- [ ] Finally 🏁 your last thing to fix is spacing between **about** and **contact** section. Please compare your design to Figma's design from the images below. 💁‍♂️ If you get stuck use this tip to make your job easier: You can add `margin-top: -150px` to the **contact** section and increase the height of **about** section 😄👌

**Your design**
![image](https://user-images.githubusercontent.com/69011963/149140072-b13ff168-3e89-499f-8e9f-b9e0ca06f47c.png)

**Figma design**
![image](https://user-images.githubusercontent.com/69011963/149140506-929dd6a2-1a90-4ddd-a238-30b9fe455558.png)